cspell.json

```diff
@@ -1582,10 +1582,15 @@
     "nextamplifygen",
     "multirepo",
     "startbuild",
+    "PITR",
+    "osis",
+    "prepper",
+    "OSIS",
```

Comment:
osis is added twice, just different capitalization. Is that intentional?

Reply:
removed `OSIS`